src/wallet/account.ts

```diff
@@ -80,41 +80,78 @@ export class WalletAccount extends Account implements AccountInterface {
   }
 
   /**
-   * WALLET EVENTS
+   * Registers a callback function to be executed when the account changes.
```

Comment:
we can remove all of these JSDOCs from just public methods, we need only on the ones that are exported...

`onAccountChange` is exported in the connect file